src/main/java/edu/aau/groupc/canteenbackend/mgmt/Canteen.java

```diff
@@ -0,0 +1,83 @@
+package edu.aau.groupc.canteenbackend.mgmt;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import edu.aau.groupc.canteenbackend.entities.DBEntity;
+import edu.aau.groupc.canteenbackend.orders.Order;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "canteen")
+public class Canteen implements DBEntity {
+
+    @Id
+    @Column(name = "CANTEEN_ID")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    @NotNull
+    private String name;
+    @NotNull
+    private String address;
+    private int numTables;
+    @OneToMany(mappedBy = "canteen")
```

Comment:
Is this not configured to be fetched lazily (i.e. no `fetch = FetchType.LAZY`) on purpose?

Reply:
Ok, nevermind! Turns out the default fetch type for `OneToMany` is already `LAZY` ([Source](https://stackoverflow.com/a/26601720)).